backend/packages/Upgrade/src/api/services/AnalyticsService.ts

```diff
@@ -183,7 +185,12 @@ export class AnalyticsService {
       let skip = 0;
       const take = 50;
       do {
-        const data = await this.analyticsRepository.getCSVDataForSimpleExport(experimentId, skip, take);
+        let data: any[];
```

Comment:
I have a request, data needs a type for sure but we're not declaring types for the return values of `getCSVDataForSimpleExport` and `getCSVDataForWithInSubExport` right now, so:

Can you update the return types for those two functions with declared types, like `Promise<CSVDataForSimpleExport[]>` and `Promise<CSVDataForWithinSubExport[]>`.

Also, please call `data` something else, because everything is data... `csvExportData` at least?

Then we can say `let csvExportData: CSVDataForSimpleExport[] | CSVDataForWithinSubExport[];`, or even create a CSVExportData parent interface that both of these extend so you can have `let csvExportData: CSVExportData`.

Reply:
resolved